code/modules/mob_spawn/corpses/job_corpses.dm

```diff
@@ -59,6 +59,11 @@
 	outfit = /datum/outfit/job/assistant
 	icon_state = "corpsegreytider"
 
+/obj/effect/mob_spawn/corpse/human/roboticist
```

Comment:
move this somewhere else so it isn't sandwiched between the Assistant type declaration and its initialize proc

Reply:
Aw dang I was blind... sorry!